cmd/sriovdp/manager.go

```diff
@@ -16,59 +16,109 @@ package main
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
+	"io"
 	"os"
+	"path/filepath"
 
 	"github.com/golang/glog"
 	"github.com/jaypipes/ghw"
 
+	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/cdi"
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/factory"
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/types"
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/utils"
 )
 
 const (
-	socketSuffix = "sock"
+	defaultConfig = "/etc/pcidp/config.json"
+	socketSuffix  = "sock"
 )
 
+// cliParams presents CLI parameters for SR-IOV Network Device Plugin
 type cliParams struct {
 	configFile     string
 	resourcePrefix string
+	useCdi         bool
+	hostConfigDir  string
 }
 
-type resourceManager struct {
+// ResourceManager manages resources for SR-IOV Network Device Plugin binaries
+type ResourceManager struct {
 	cliParams
 	pluginWatchMode bool
 	rFactory        types.ResourceFactory
-	configList      []*types.ResourceConfig
+	ConfigList      []*types.ResourceConfig
 	resourceServers []types.ResourceServer
 	deviceProviders map[types.DeviceType]types.DeviceProvider
 }
 
-func newResourceManager(cp *cliParams) *resourceManager {
+// FlagInit parse command line flags
```

Comment:
do we really need to move this one ?

Reply:
Generally i think its good to move resource manager out of main to its own package under pkg, WDYT ?

we are more than half way there. ...